src/main/java/org/opentripplanner/routing/alertpatch/TransitAlert.java

```diff
@@ -119,6 +121,10 @@ public ZonedDateTime creationTime() {
     return creationTime;
   }
 
+  public Integer version() {
```

Comment:
Add JavaDoc here. Important so specify that this is only supported by Siri, not GTFS RT.